mail-filter/pyzor/pyzor-1.0.1_pre20200523.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+inherit distutils-r1
+
+COMMIT="f46159bd6519cebcebf59e9334a7920371111d75"
+
+DESCRIPTION="A distributed, collaborative spam detection and filtering network"
+HOMEPAGE="https://github.com/SpamExperts/pyzor"
+SRC_URI="https://github.com/SpamExperts/pyzor/archive/${COMMIT}.tar.gz -> ${PN}-${COMMIT}.tar.gz"
```

Comment:
-> ${P} is usually preferred

Reply:
Usually yes, but this is a pre-release, the version can change. The git commit makes it unique and avoids collision.